korim/src/commonMain/kotlin/com/soywiz/korim/format/ASE.kt

```diff
@@ -39,6 +31,10 @@ import com.soywiz.korma.geom.RectangleInt
 import com.soywiz.korma.geom.slice.*
 import com.soywiz.krypto.encoding.hex
 
+// If this is true, only processes visible layers from the ASE file.
+// Otherwise, will process invisible layers as well.
+var ImageDecodingProps.onlyReadVisibleLayers: Boolean by extraProperty { true }
```

Comment:
This is much more elegant than the getExtra() properties approach for "layers", "disableSlicing" and "useSlicePosition". Those are used in ParallaxDataView when reading from aseprite file.
Maybe I can align that later :)